packages/auth/src/next/helpers/user.ts

```diff
@@ -1,31 +1,23 @@
+import type { NextRequest } from "next/server";
+import { cookies, headers } from "next/headers";
+
 import {
   THIRDWEB_AUTH_ACTIVE_ACCOUNT_COOKIE,
   THIRDWEB_AUTH_TOKEN_COOKIE_PREFIX,
 } from "../../constants";
 import { Json } from "../../core";
-import { ThirdwebAuthContext, ThirdwebAuthUser } from "../types";
-import { GetServerSidePropsContext, NextApiRequest } from "next";
-import { NextRequest } from "next/server";
-
-export function getCookie(
-  req: GetServerSidePropsContext["req"] | NextRequest | NextApiRequest,
-  cookie: string,
-): string | undefined {
-  if (typeof req.cookies.get === "function") {
-    return req.cookies.get(cookie);
-  }
+import {
+  ThirdwebAuthContext,
+  ThirdwebAuthUser,
+} from "../types";
 
-  return (req.cookies as any)[cookie];
+export function getCookie(cookie: string): string | undefined {
```

Comment:
Instead of supporting just the app router format, we need to support both the app router & the old pages directory - so we need to check for both cases here